__snapshots__/default-changelog-notes.ts.js

```diff
@@ -52,6 +52,15 @@ exports['DefaultChangelogNotes buildNotes should handle BREAKING CHANGE notes 1'
 * some bugfix ([sha2](https://github.com/googleapis/java-asset/commit/sha2))
 `
 
+exports['DefaultChangelogNotes buildNotes should ignore "chore: release" commits 1'] = `
+## [1.2.3](https://github.com/googleapis/java-asset/compare/v1.2.2...v1.2.3) (1983-10-10)
+
+
+### Chores
+
+* some chore ([sha1](https://github.com/googleapis/java-asset/commit/sha1))
+`
+
```

Comment:
@RobertCraigie Could you confirm that looks as expected?

Reply:
yeah looks great, thanks!